js/scenes/Login/LoginContainer.js

```diff
@@ -36,10 +36,13 @@ class LoginContainer extends Component {
         password: this.props.passwordField
       }
       try {
-        const success = await login(loggedInUser)
-        if (success) {
-          this.loginRedirect()
+        login(loggedInUser).then((res, rej) => {
+          if (res) { this.loginRedirect() }
```

Comment:
If you're not using the arguments, you don't need to return them.  There's also no need to have an `if` and `else` statement inside `then`,  if it got there, it was successfully executed otherwise, it would've gotten into the `catch`